.build

```diff
@@ -1,4 +1,5 @@
 CC = gcc
```

Comment:
Thank! I really appreciate your taking a look at this!

You can go ahead and commit this PR if you want, the other change can come later, OR you can make the change now. I leave it in your hands :-)